OneSignalSDK/onesignal/src/main/java/com/onesignal/OSViewUtils.java

```diff
@@ -91,11 +91,12 @@ else  if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP)
     private static int getWindowHeightAPI23Plus(@NonNull Activity activity) {
         View decorView = activity.getWindow().getDecorView();
         // Use use stable heights as SystemWindowInset subtracts the keyboard
-        int heightWithoutKeyboard =
-           decorView.getHeight() -
-           decorView.getRootWindowInsets().getStableInsetBottom() -
-           decorView.getRootWindowInsets().getStableInsetTop();
-        return heightWithoutKeyboard;
+        if (decorView.getRootWindowInsets() == null)
```

Comment:
Let's save `decorView.getRootWindowInsets()` into a variable just in case it goes `null` in between calls.

Reply:
Very true, thanks